build.js

```diff
@@ -54,6 +55,10 @@ const createZipFile = async (fileName, path) => {
     const version = await getVersion();
 
     for (const browser in BROWSERS) {
+        if (params.includes('--safari-web-extension') && browser != 'Safari') {
```

Comment:
Use strict comparisons everywhere. `!=` should be `!==` and `==` should be `===` etc.